### ISSUE TITLE
Ability To Minimize MediaElch While Scanning Files

### ISSUE THREAD

**Issue by lynxstrike** [Komet/MediaElch#973]
**Is your feature request related to a problem? Please describe.**
It wold be really nice if one could minimize MediaElch's windows while it's working scanning for files. It's a little frustrating to have to keep the window open while it is working.
**Describe the solution you'd like**
Ability To Minimize MediaElch While Scanning Files.
**Describe alternatives you've considered**

**Additional context**

**Comment by bugwelle** [comment]
Interesting. You mean after clicking the "Reload" button you're no longer able to minimize MediaElch?

**Comment by lynxstrike** [comment]
Yes... it has to stay put on the screen... it's a little messy... while if you are able to minimize it to the taskbar you can continue to work on other things while MediaElch works... :)

**Comment by bugwelle** [comment]
Ok, this is either specific to your system (was it Windows?) or I've just never noticed it. :o

**Comment by lynxstrike** [comment]
As far as I know (because I only work with windows) this has to be a Windows in genera issue: the little message window "scanning files" does not allow to press the minimize button on the MediaElch main program window.

My system:

Windows 10 x64 pro, latest
MideaElch stand alone latest

**Comment by bugwelle** [comment]
Ah, thanks. Yes, that Dialog Window is a popup. It should be possible to minimize it if I add

```cpp
setWindowFlags(windowFlags() | Qt::WindowMinimizeButtonHint);
```

But I have to test it. :-)

**Comment by chchia** [comment]
instead of minimizing it, i really hope that MediaElch can allow to work on existing database while it is scanning for update rather then locking the whole GUI and behaving like "hang".

just for your info, i am also using TinyMediaManager, and TMM allow on the fly update the database while scanning, and you can manipulated the scanned file immediately after it is added to database even though scanning is still in process. this will be a +++ for MediaElch.
